cumulusci/cli/flow.py

```diff
@@ -29,7 +29,7 @@ def flow_doc(runtime):
     click.echo(flow_ref_title_and_intro(flow_info["intro_blurb"]))
     flow_info_groups = list(flow_info["groups"].keys())
 
-    flows = runtime.get_available_flows()
+    flows = runtime.get_available_flows(project=False)
```

Comment:
Hmm, previously it was possible to run `cci flow doc` in a CumulusCI project to get output including project-specific flows. I know that's not what we want for CumulusCI's own documentation, but I'm not sure we should remove that capability for other projects. It at least warrants a release note.

Reply:
Would a good middle ground be to provide the `cci flow doc` command a `--project` flag? When passed, it would also include documentation for project specific flows?